index.test.js

```diff
@@ -216,6 +216,20 @@ describe('validate-commit-msg.js', function() {
       expect(errors).to.deep.equal(['INVALID COMMIT MSG: subject does not match subject pattern!']);
       expect(logs).to.deep.equal([msg]);
     });
+
+    it('should lowercase type when autoFix is true and make it valid', function() {
+      m.config.autoFix = true;
```

Comment:
Yes, they could use refactoring :) I'm no longer in favor of using `beforeEach`/`afterEach` or nesting in general if it can be avoided (which it often can).